docker-compose.yml

```diff
@@ -43,3 +43,14 @@ services:
       MYSQL_ROOT_HOST: 0.0.0.0
     volumes:
       - ./test/mysqlConfig:/etc/mysql/conf.d
+
+  postgres:
+    image: postgres:10.4
+    ports:
+      - 5432:5432
+    environment:
+      POSTGRES_USER: node
+      POSTGRES_PASSWORD: nodepw
+      POSTGRES_DB: nodedb
+    volumes:
+      - /srv/docker/postgresql:/var/lib/postgresql
```

Comment:
What's that `volume` for? This fails for me locally. I guess it can be removed, right?

Reply:
That should give persistent data.  I'll check if we really need it.